### DISCUSSION TITLE
`pip install py7zr` does not use wheel  but use source archive on windows 11

### DISCUSSION THREAD

**Discussion by Atlanta11** [miurahr/py7zr#565] [Q&A]
After installing [https://www.python.org/ftp/python/3.10.5/python-3.10.5-amd64.exe](url) on **windows 11** and then with cmd start **pip install py7zr** it will not install further.
![Schermafbeelding 2022-07-01 165529](https://user-images.githubusercontent.com/9640147/176918890-bd03f88a-96d0-4372-9b34-2f74b6290a9e.png)

Please help.

**Comment by miurahr** [comment]
Thank you for report.

We release ``py7zr`` with source and wheels, but v0.18.5-v0.18.9 released only source because of release script broken.
Now v0.18.10 released with wheel binary package.

**Comment by Atlanta11** [comment]
Oke after reinstalling end restarting it works.
Is there also a way if I zip a large file so I can see the progress with a progress bar?
**Thanks in advance.**

**Comment by miurahr** [comment]
Your question is similar to #460 
It has an undocumented API but CLI does not support it yet.

**Comment by miurahr** [comment]
@Atlanta11 does new release, v0.18.10 or v0.19.0, change the behavior?

**Comment by miurahr** [comment]
resolved.
